packages/alfa-url/src/url.ts

```diff
@@ -34,10 +35,25 @@ export class URL implements Equatable, Hashable, Serializable<URL.JSON> {
       port,
       Sequence.from(path),
       query,
-      fragment
+      fragment,
+      cannotBeABase
     );
   }
 
+  /**
+   * @see https://tools.ietf.org/html/rfc2606#section-3
+   */
+  public static example(): URL {
+    return URL.parse("https://example.com").get();
+  }
+
+  /**
+   * @see https://tools.ietf.org/html/rfc6694#section-3
+   */
+  public static blank(): URL {
+    return URL.parse("about:blank").get();
+  }
+
```

Comment:
These two seem to be unused 🤔

Reply:
`URL.blank()` currently is, but `URL.example()` isn't.